lib/node/closure-compiler-js.js

```diff
@@ -25,6 +25,7 @@
 const path = require('path');
 const contribPath = path.dirname(path.resolve(__dirname, '../../')) + '/contrib';
 const jscomp = require('../../jscomp.js');
+const CONSOLE_COLOR_CHARS = /\u001B\[\d+m/ug;
```

Comment:
[ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code#3/4_bit) seems to suggest that

```js
    /\u001B\[\d+(?:[:;]\d+)*m/g
```

might match more styles though I have no idea what subset of that JSCompiler actually uses.

Reply:
It only needs to match these: https://github.com/google/closure-compiler-npm/blob/master/lib/logger.js#L23-L27